### PR TITLE
Update ReadMe images

### PR DESCRIPTION
**Pull request by ElviaBth**
🎩 What? Why?
Due to the upgrade 0.28, features have changed so there is a need to update ReadMe images as well. 

📌 Related Issues
#270 Upgrade 0.28

📷 Screenshots

♥️ Thank you!

### CODE REVIEW

File: examples/custom-redirections.png
Comment (microstudi) [inline examples/custom-redirections.png:1]: use https://decidim-awesome.dev.pokecode.net/ for this screenshot

File: examples/menu-3.png
Comment (microstudi) [inline examples/menu-3.png:1]: we should see a change here, you need to use the content block menu hack to tune this menu

File: examples/proposal-images.png
Comment (microstudi) [inline examples/proposal-images.png:1]: in here, better put the same text by hand

File: examples/quill-images.png
Comment (microstudi) [inline examples/quill-images.png:1]: a nice image better

File: image-1.png
Comment (microstudi) [inline image-1.png:1]: remove?

File: image.png
Comment (microstudi) [inline image.png:1]: remove?

File: examples/custom-redirections.png
Comment (ElviaBth) [reply examples/custom-redirections.png:1]: Done

File: examples/proposal-images.png
Comment (ElviaBth) [reply examples/proposal-images.png:1]: Done

File: examples/quill-images.png
Comment (ElviaBth) [reply examples/quill-images.png:1]: Done

File: image.png
Comment (ElviaBth) [reply image.png:1]: Done

File: image-1.png
Comment (ElviaBth) [reply image-1.png:1]: Done

File: examples/custom-redirections.png
Comment (ElviaBth) [reply examples/custom-redirections.png:1]: Done

File: examples/menu-3.png
Comment (ElviaBth) [reply examples/menu-3.png:1]: Ready

**Pull request merged**